lib/ui/widget/keypad/key_pad_buttons.dart

```diff
@@ -7,16 +7,21 @@ import 'package:stacked/stacked.dart';
 class KeyPadButtons extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Wrap(
-      // alignment: Alignment.center,
-      children: _buildButtons(),
-    );
+    return ViewModelBuilder.reactive(
+        builder: (BuildContext context, PayableViewModel model, Widget child) {
+          return Wrap(
+            // alignment: Alignment.center,
+            children: _buildButtons(model.showButton),
+          );
+        },
+        viewModelBuilder: () => PayableViewModel());
   }
 
-  List<Widget> _buildButtons() {
+  List<Widget> _buildButtons(bool showButton) {
```

Comment:
Let's use named parameter like {bool showButton} this is much easy when someone is calling a function to know what are parameters.

Reply:
Let's move this bool showButton to a constructor so it can when we call KeyPadButtons we can be able to pass it that variable.